data/mods/Aftershock/maps/overmap_map_extras.json

```
@@ -18,5 +18,27 @@
     "sym": "b",
     "color": "pink",
     "autonote": true
+  },
+  {
+    "id": "mx_gunship",
+    "type": "map_extra",
+    "name": { "str": "Gunship Crash" },
+    "description": "Gunship crashed here.",
+    "generator": { "generator_method": "map_extra_function", "generator_id": "mx_gunship" },
+    "min_max_zlevel": [ 0, 0 ],
+    "sym": "X",
+    "color": "light_blue",
+    "autonote": true
+  },
+  {
+    "id": "mx_truth_reconciliation",
+    "type": "map_extra",
+    "name": { "str": "Gunship Crash" },
+    "description": "Gunship crashed here.",
+    "generator": { "generator_method": "map_extra_function", "generator_id": "mx_truth_reconciliation" },
+    "min_max_zlevel": [ 0, 0 ],
+    "sym": "X",
+    "color": "light_blue",
+    "autonote": true
   }
```

```suggestion
  {
    "id": "mx_gunship",
    "type": "map_extra",
    "name": { "str": "Gunship Crash" },
    "description": "Gunship crashed here.",
    "generator": { "generator_method": "update_mapgen", "generator_id": "mx_gunship" },
    "min_max_zlevel": [ 0, 0 ],
    "sym": "X",
    "color": "light_blue",
    "autonote": true
  },
  {
    "id": "mx_truth_reconciliation",
    "type": "map_extra",
    "name": { "str": "Gunship Crash" },
    "description": "Gunship crashed here.",
    "generator": { "generator_method": "update_mapgen", "generator_id": "mx_truth_reconciliation" },
    "min_max_zlevel": [ 0, 0 ],
    "sym": "X",
    "color": "light_blue",
    "autonote": true
  }
```
`map_extra_function` means it's looking for a function in C++ to place these